src/app/build.gradle

```diff
@@ -1,22 +1,7 @@
-import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
-
 buildscript {
 	ext.kotlin_version = '1.3.30'
 }
 
-plugins {
```

Comment:
So everything red here I just moved to being declared in the "subprojects" block of the root project